src/authenticator.ts

```diff
@@ -11,6 +11,10 @@ export interface TokenWithExpire {
   expire: number;
 }
 
+export interface RefreshToken {
+  value: string;
```

Comment:
I wonder whether these objects should have the same key name for consistency. At the moment one has `token` and one has `value`. What about calling it something like `rawToken`, or `tokenString`, or just in both? I don't feel too strongly about this, so it's up to you.